pkg/lib/cockpit-components-dropdown.tsx

```diff
@@ -36,7 +35,14 @@ import { EllipsisVIcon } from '@patternfly/react-icons';
  * require a separator between DropdownItem's use PatternFly's Divivder
  * component.
  */
-export const KebabDropdown = ({ dropdownItems, position = "end", isDisabled = false, toggleButtonId, isOpen, setIsOpen, props }) => {
+export const KebabDropdown = ({ dropdownItems, position = "end", isDisabled = false, toggleButtonId, isOpen, setIsOpen, props } : {
+    dropdownItems: React.ReactNode,
+    position?: "start" | "end" | "right" | "left" | "center",
+    isDisabled?: boolean,
+    toggleButtonId?: string;
+    isOpen?: boolean, setIsOpen?: React.Dispatch<React.SetStateAction<boolean>>,
+    props?: Parameters<typeof Dropdown>[0]
```

Comment:
This is afaik unused and should rather be dropped.

Reply:
There's indeed no user of this in the Cockpit repo.

Reply:
... or any other repo.